locales/en-US/snoozetabs.properties

```diff
@@ -62,34 +62,21 @@ manageUndo=Undo
 # Firefox opens
 manageDateNextOpen=Next time<br>Firefox opens
 
-calendar_today=Today
-calendar_now=Now
-calendar_backToToday=Back to today
-calendar_ok=Ok
-calendar_clear=Clear
-calendar_month=Month
-calendar_year=Year
-calendar_timeSelect=Select time
-calendar_dateSelect=Select date
+# Tooltip label for the table header in date picker that can be clicked to select a month.
 calendar_monthSelect=Choose a month
+# Tooltip label for the table header in date picker that can be clicked to select a year.
 calendar_yearSelect=Choose a year
+# Tooltip label for the table header in date picker that can be clicked to select a decade.
 calendar_decadeSelect=Choose a decade
-calendar_yearFormat=YYYY
-calendar_dateFormat=M/D/YYYY
-calendar_dayFormat=D
-calendar_dateTimeFormat=M/D/YYYY HH:mm:ss
-calendar_monthFormat=MMMM
-# true if the month name should appear before the year when displayed, false if not
-calendar_monthBeforeYear=true
-# (PageUp) is a description of the keyboard shortcut
+# Tooltip label for the arrow button in date picker to move to the previous month. (PageUp) is a description of the keyboard shortcut
 calendar_previousMonth=Previous month (PageUp)
-# (PageDown) is a description of the keyboard shortcut
+# Tooltip label for the arrow button in date picker to move to the next month. (PageDown) is a description of the keyboard shortcut
 calendar_nextMonth=Next month (PageDown)
-# (Control + left) is a description of the keyboard shortcut
-calendar_previousYear=Last year (Control + left)
```

Comment:
We still need this (and its sibling), it's displayed when you click on the month name.

```
# Tooltip for the arrow to move to the previous year in month selection
```